packages/react-scripts/config/tslint.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
Let's put this in a package, `tslint-config-react-app` or `tslint-react-app` (whichever follows TSLint convention).

Reply:
Done!